pegasus/sites.v3/code.org/public/js/about_headshots.js

```diff
@@ -0,0 +1,32 @@
+window.addEventListener('resize', function () {
+    window.location.reload();
+});
+
+if (document.documentElement.clientWidth < 970) {
+  $('.avatar_container').outerHeight();
+} else {
+  $(window).load(function () {
+
+    let i = 2;
+    let j = 3;
+    let k = 4;
+
+    while ( i < 63 ) {
```

Comment:
There are some magic numbers here (what is 63?) which should be hoisted out into named constants.

Is 63 the number of staff?  We should be calculating that so that we can adapt to new staff being added via the gsheet.